modules/bases2fastq/main.nf

```diff
@@ -0,0 +1,45 @@
+process BASES2FASTQ {
+    tag "$meta.id"
+    label 'process_high'
+
+    if (params.enable_conda) {
+        exit 1, "Conda environments cannot be used when using bases2fastq. Please use docker or singularity containers."
+    }
+    container "elembio/bases2fastq:1.1.0"
+
+    input:
+    tuple val(meta), path(run_manifest), path(run_dir)
```

Comment:
Nitpicking, but since `run_manifest` doesn't seem to be mandatory wouldn't it be more "code-convention" to have that argument last?

Reply:
I agree, but I was trying to follow the same structure as your `bclconvert` module. I'd agree with it being last.